MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/EnggDiffraction/IEnggDiffractionView.h

```diff
@@ -387,6 +387,13 @@ class IEnggDiffractionView {
   virtual std::string fittingPeaksData() const = 0;
 
   /**
+  * Sets the peak list according to the QString given
+  *
+  * @param peakList list of expected peaks to be fitted as QString
+  */
+  virtual void setPeakList(QString peakList) const = 0;
```

Comment:
Just one small detail. This is an interface which does not use Qt classes for anything. It would be better to use `std::string` rather than adding `QString`, `QStringList`, etc. here. As the presenter is preparing the strings as `std::string`, this fits well between the view and the presenter. Only the Qt specific view classes should use Qt classes, as far as possible.